eng/common/templates/job/job.yml

```diff
@@ -136,7 +136,7 @@ jobs:
         condition: and(succeeded(), in(variables['_SignType'], 'real', 'test'), eq(variables['Agent.Os'], 'Windows_NT'))
 
   - ${{ if and(eq(parameters.runAsPublic, 'false'), eq(variables['System.TeamProject'], 'internal')) }}:
-    - task: NuGetAuthenticate@0
+    - task: NuGetAuthenticate@1
```

Comment:
These files (in eng/common) are managed by arcade and will get overwritten by the next arcade update.  We may already have an automated arcade update fixing this, or we need to get one from them.

Reply:
Let me see if the arcade updated is ignored somehwere

Reply:
Is this supposed to update the task? I don't see it's change this  https://github.com/dotnet/roslyn/pull/71604